.github/workflows/deploy.yml

```diff
@@ -0,0 +1,89 @@
+name: Deploy to TT02 and Production
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: environment
+        description: Select the environment
+      version:
+        type: string
+        description: Enter the package version
+        required: true
+          
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: altinn/altinn-auth-audit-log:${{ github.event.release.tag_name }}
+  CONTAINER_APP_CONTAINER_NAME: auditlog
+  
+jobs:
+  deploy-tt02:
+    environment: TT02
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+      packages: read
+    steps:
+    - name: Log in to the Container registry
+      uses: docker/login-action@v3
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - name: Azure Login
+      uses: azure/login@v1
+      with:
+        client-id: ${{ secrets.AZURE_CLIENT_ID }}
+        tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+        subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+    - name: Deploy to containerapp
+      uses: azure/CLI@v1
+      with:
+        inlineScript: >
+          az config set extension.use_dynamic_install=yes_without_prompt
+          
+          az containerapp update \
+            --name ${{ vars.CONTAINER_APP_NAME }} \
+            --container-name ${{ env.CONTAINER_APP_CONTAINER_NAME }} \
+            --resource-group ${{ vars.CONTAINER_APP_RESOURCE_GROUP_NAME }} \
+            --image ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+    - name: logout
+      run: >
+        az logout
+
+  deploy-prod:
+    environment: PROD
```

Comment:
This job should run after deploy-tt02, so it needs a "needs" clause

Reply:
fixed